TACC/Core-Styles
# fix: DES-2724 app-card__type mis-aligned

## Overview

Fix mis-aligned "Types" block on an app card.

_The mis-alignment is not always present, but once I could reproduce it, this made sense to fix it._

## Related

- found while testing [DES-2724](https://tacc-main.atlassian.net/browse/DES-2724)

## Changes

- **changed** margin bottom on card `__types` box to a static value

## Testing

1. Visit https://www.designsafe-ci.org/testing-pages/image-plugin-email-link/.
2. Verify gray boxes still line up.
3. Remove font size of text in gray box*:
    1. find `c-app-card__type` styles
    2. disable `font-size: 1.2rem`
4. Verify gray boxes still line up.

<sup>* There may be other ways to product the bug. This worked for me.</sup>

## UI

| | Bug **is** apparent | Bug **not** apparent |
| - | - | - |
| Before | <img width="1194" alt="bug is apparent" src="https://github.com/TACC/Core-Styles/assets/62723358/32b55ae6-8538-4b70-809c-e0413e731334"> | <img width="1194" alt="bug not apparent" src="https://github.com/TACC/Core-Styles/assets/62723358/6f3a3afc-6d4b-417a-a827-5bfd0de53e6d"> |
| After | …
